src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -357,8 +357,70 @@ now be added using the *Create Node Source* panel in the Resource Manager or the
     $ PROACTIVE_HOME/bin/proactive-client --createns vmware -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.VmwareInfrastructure username password endpoint rmHostname connectorIaasURL image ram cores vmusername vmpassword numberOfInstances numberOfNodesPerInstance downloadCommand additionalProperties
 
 
-WARNING: When the Scheduler is stopped (without removing the created infrastructure), the VmWare instances will not be terminated. And when the Scheduler is restarted, the infrastrucutre will be re-configured as per previous settings. If the instances were delete at the VmWare server side, they will be re-created and re-configured.
+WARNING: When the ProActive server is stopped (without removing the created infrastructure), the VmWare instances will not be terminated. And when the ProActive server is restarted, the infrastrucutre will be re-configured as per previous settings. If the instances were delete at the VmWare server side, they will be re-created and re-configured.
```

Comment:
Please replace 'the ProActive server' simply with 'ProActive server' in the whole doc
replace  remove **the**  in sentences such as 'the VmWare instances' in the whole doc
add **the** before 'previous settings'.
replace **delete** with **deleted** in the whole doc